integration_test/index_test.go

```diff
@@ -38,7 +39,7 @@ func TestKrewIndexAdd(t *testing.T) {
 	if err := test.Krew("index", "add", "../../usr/bin", constants.IndexURI); err == nil {
 		t.Fatal("expected index add with path characters to fail")
 	}
-	if err := test.Krew("index", "add", "foo", constants.IndexURI).Run(); err != nil {
+	if err := test.Krew("index", "add", "foo", test.TempDir().Path("index/"+constants.DefaultIndexName)).Run(); err != nil {
 		t.Fatalf("error adding new index: %v", err)
 	}
```

Comment:
can you add repeat this command but test "re-adding existing index must fail"

Reply:
Ah that's a good idea.